### PR TITLE
fix #5825: add copy feature to controls

### LINKED ISSUE #5825: add copy feature to controls
Add a Copy button to the list of options when you hover over a control command. Similar to the Copy feature already available on the Lattice tab.

### PR DESCRIPTION
**Pull request by rorour**

### CODE REVIEW

File: sirepo/package_data/static/js/sirepo-command.js [hunk 1]
```diff
@@ -82,6 +82,15 @@ SIREPO.app.factory('commandService', function(appState, latticeService, panelSta
         return value;
     };
 
+    self.getNextCommand = (type) => {
```
Comment (mkeilman) [inline sirepo/package_data/static/js/sirepo-command.js:85]: I think we are omitting parens for single parameters:
```javascript
self.getNextCommand = type => {...};
```

File: sirepo/package_data/static/js/sirepo-command.js [hunk 2]
```diff
@@ -242,6 +251,11 @@ SIREPO.app.directive('commandTable', function(appState, commandService, latticeS
                 return -1;
             }
 
+            $scope.copyCommand = function(cmd) {
```
Comment (mkeilman) [inline sirepo/package_data/static/js/sirepo-command.js:254]: As above, use arrow notation and omit parens:
```javascript
scope.copyCommand = cmd => {...}
```

File: sirepo/package_data/static/js/sirepo-command.js [hunk 3]
```diff
@@ -242,6 +251,11 @@ SIREPO.app.directive('commandTable', function(appState, commandService, latticeS
...
+                const copy = $.extend({}, commandService.commandForId(cmd._id), commandService.getNextCommand(cmd._type));
```
Comment (mkeilman) [inline sirepo/package_data/static/js/sirepo-command.js:255]: FYI javascript supports using the spread operator for this so you don't have to rely on jquery:
```javascript
const copy = {...commandService.commandForId(cmd._id), ...commandService.getNextCommand(cmd._type))};
```
You can also skip the temp variable

**Review by mkeilman — COMMENTED** [review]
Sorry to jump in - just a couple of comments

File: sirepo/package_data/static/js/sirepo-command.js (hunk 3 above)
Comment (rorour) [reply sirepo/package_data/static/js/sirepo-command.js]: thanks! 77045d74288ceb536b10500920e635863ad3d399

File: sirepo/package_data/static/js/sirepo-command.js (hunk 2 above)
Comment (rorour) [reply sirepo/package_data/static/js/sirepo-command.js]: 77045d74288ceb536b10500920e635863ad3d399

File: sirepo/package_data/static/js/sirepo-command.js (hunk 1 above)
Comment (rorour) [reply sirepo/package_data/static/js/sirepo-command.js]: 77045d74288ceb536b10500920e635863ad3d399

**Review by mkeilman — APPROVED** [review]
Looks good!

**Pull request merged**